tests/snappi_tests/ecn/test_red_accuracy_with_snappi.py

```diff
@@ -63,10 +63,10 @@ def test_red_accuracy(request,
     snappi_extra_params = SnappiTestParams()
     snappi_extra_params.packet_capture_type = packet_capture.IP_CAPTURE
     snappi_extra_params.is_snappi_ingress_port_cap = True
-    snappi_extra_params.ecn_params = {'kmin': 500000, 'kmax': 2000000, 'pmax': 5}
+    snappi_extra_params.ecn_params = {'kmin': 500000, 'kmax': 900000, 'pmax': 5}
```

Comment:
pls seperate this

Reply:
@selldinesh Please take care of this comment